### PR TITLE
PWN-6722 - Returning back unit tests and testing release pull requests

### PR DESCRIPTION
**Pull request by 1sakov**
Testing CI/CD

### CODE REVIEW

**Review by Davran331122 — APPROVED** [review]

**Comment by Davran331122** [comment]
@1sakov test job was failed x D I added WIP label for your MR

File: app/src/main/java/org/p2p/wallet/common/analytics/trackers/AppsFlyerTracker.kt [hunk 1]
```diff
@@ -6,10 +6,10 @@ import com.appsflyer.attribution.AppsFlyerRequestListener
 import timber.log.Timber
 
 class AppsFlyerTracker(
-    private val context: Context
+    private val context: Context,
+    private val appsFlyer: AppsFlyerLib
 ) : AnalyticsTracker {
 
-    private val appsFlyer = AppsFlyerLib.getInstance()
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/common/analytics/trackers/AppsFlyerTracker.kt:12]: Why did you move it outside?
